app/pages/home-not-logged-in.cjsx

```diff
@@ -32,30 +32,31 @@ module.exports = React.createClass
             <source type="video/mp4" src="./assets/home-video.mp4"></source>
           </video>
         }
-
-        <h1>THE ZO<ZooniverseLogo />NIVERSE</h1>
-        <h2 className="lighter">is people powered research</h2>
-        <h2>
-          <Link to="/about" className="standard-button">Learn More</Link>
-        </h2>
+        <div className="home-intro-content">
+          <h1>THE ZO<ZooniverseLogo />NIVERSE</h1>
+          <h2 className="lighter">is people powered-research</h2>
+          <div className="home-intro-buttons">
+            <button className="button standard-button" onClick={@scrollDown}>Learn More</button>
+            <Link to="/projects" className="button intro-button">Get Started</Link>
+          </div>
+        </div>
+        <button className="home-link-down" onClick={@scrollDown}>
+          <i className="fa fa-angle-down" onClick={@scrollDown} />
+        </button>
       </section>
 
-      <button className="home-link-down" onClick={@scrollDown}>
-        <i className="fa fa-angle-down" onClick={@scrollDown} />
-      </button>
-
       <section className="home-discover" ref="discover">
         <h1>Discover, teach, and learn</h1>
 
         <p>
-          The Zooniverse enables everyone to take part in real cutting edge
+          The Zooniverse enables everyone to take part in real cutting-edge
           research in many fields across the sciences, humanities, and more.
           The Zooniverse creates opportunities for you to unlock answers and
           contribute to real discoveries.
         </p>
 
         <h2>
-          <Link to="/projects" className="standard-button">Get started</Link>
+          <Link to="/projects" className="button discover-button">Choose a Project</Link>
```

Comment:
You can drop the extra class name and just specify the css as `.home-discover .standard-button` or something similar